includes/free-trial/class-wc-calypso-bridge-free-trial-plan-picker-banner.php

```diff
@@ -36,9 +36,11 @@ public function __construct() {
 
 		add_action('init', function() {
 			if ( current_user_can( 'manage_woocommerce' ) ) {
-				add_action( 'wp_footer', array( $this, 'add_plan_picker_banner' ), 10 );
+				add_action( 'wp_head', array( $this, 'add_plan_picker_banner' ), -2000 );
```

Comment:
In a normal situation, `wp_body_open` should be used instead. However, `wpcomsh` uses [wp_head](https://github.com/Automattic/wpcomsh/blob/trunk/private-site/private-site.php#L102) hook to render the launch banner (screenshot). In order to render the free trial banner before the launch bar, we also have to use `wp_head` with higher priority.

FYI, when a browser detects an HTML element in the head tag, it actually renders the element in the body correctly.

![Screen Shot 2023-04-05 at 11 10 39 AM](https://user-images.githubusercontent.com/4723145/230167534-87ff26b4-25e5-43d3-958d-4504ea3af2d8.jpg)